app/Console/Commands/Tests/SetupDummyAccount.php

```diff
@@ -242,6 +245,9 @@ private function createFirstUser(): void
             'last_name' => 'Scott',
         ]);
 
+        $user->email_verified_at = Carbon::now();
```

Comment:
Right, I've missed that one. you can also set  `MAIL_VERIFY=false` in you `.env` file

Reply:
OK, i’ll also do it in a separate PR.